wordpress-mobile/WordPress-Android
# Add domains button to the me screen

Fixes #19066

### Description

This PR adds a "Domains" button to the "Me" screen, hidden by default. This can be enabled via [the testing steps in the corresponding feature flag PR](https://github.com/wordpress-mobile/WordPress-Android/pull/19203#issue-1900384857).

To test:

Enable the feature flag, and navigate to the "Me" screen. Expect to see the "Domains" button.

Note: since the domain management screens are still in development, this is not wired up yet to launch the domain management activity.

## Regression Notes
1. Potential unintended areas of impact
Me screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A (declarative UI element hidden by default - a unit test would merely be testing the `visibility` flag).

PR submission checklist:
